src/DistortableCollection.js

```diff
@@ -218,11 +218,17 @@ L.DistortableCollection = L.FeatureGroup.extend({
 
     json.images = json.images.reverse();
     json.avg_cm_per_pixel = this._getAvgCmPerPixel(json.images);
-
+    var jsonImages = json.images;
+    savetoLocalStorage(jsonImages);
```

Comment:
Actually wait, cancel that. My mistake. I meant https://github.com/publiclab/Leaflet.DistortableImage/pull/1161 -- where we are working with the `generateExportJson()` function. Unfortunately I think there is some overlap between #1237 and #1161, and I'd like to ask you and @leilayesufu to try to work together on #1237 and to record a joint contribution. Would you both be willing to try that, since it's quite a complex one? If so, you can mark the PR as collaborative and work on it together.

Reply:
> Hi, this is looking like a good start, thank you! I wanted to suggest that you build on #1237, because it is almost the same! We'll want to make a function which runs each time there is an update to an image... it could even be each time there's a MouseUp, maybe? And it'll write a new "saved state" into localStorage, and similarly to #1237, not just the selected ones but all the images.
> 
> Shall we wait a little bit until #1237 is resolve then? Thanks!

Hi Jeff, I'm little bit confused how to get started on this. Could you drop some hints? Thanks!

Reply:
Hi Jeff, just a little hint, please

Reply:
Hi @vanithaak -- so, I had realized that I was also working with @leilayesufu on a parallel track on parts of this, in https://github.com/publiclab/Leaflet.DistortableImage/pull/1161 -- and we had found that the problem was it only downloaded images that have been selected, so we needed to modify `generateExportJson()` to have a an `onlySelected` parameter defaulting to true, but so that we can override it if we want ALL images:

https://github.com/publiclab/Leaflet.DistortableImage/pull/1161#issuecomment-1296035190

Once this modification is made, we can use `generateExportJson()` for both downloading a JSON file, and storing the JSON file in localStorage. If you or @leilayesufu are interested in continuing to work on this, I think that's the key. It could even be done as its own PR -- and then we can begin using `generateExportJson(false)` to collect ALL images, for saving in localStorage OR for download. Does that make sense?